README.md

```diff
@@ -22,25 +22,18 @@ metadata in simple file formats, which is fully compatible with DVC.
 
 ______________________________________________________________________
 
-# Quickstart (Google Colab)
-
-<p align='center'>
-<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Quickstart.ipynb">
-Python API Overview
-</a>
-</p>
-
-<p align='center'>
-<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-PyTorch-Lightning.ipynb">
-PyTorch Lightning Example
-</a>
-</p>
-
-<p align='center'>
-<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-YOLO.ipynb">
-Ultralytics YOLO v8 Example
-</a>
-</p>
+# Quickstart
+
+| Python API Overview | PyTorch Lightning | Scikit-learn | Ultralytics YOLO v8 |
+|--------|--------|--------|--------| |
+<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-Quickstart.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" /></a>
+|
+<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-PyTorch-Lightning.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" /></a>
+|
+<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLIve-scikit-learn.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" /></a>
+|
+<a href="https://colab.research.google.com/github/iterative/dvclive/blob/main/examples/DVCLive-YOLO.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" /></a>
+|
```

Comment:
This is how I'm seeing it rendered:

<img width="1039" alt="Screenshot 2023-08-07 at 10 54 44 AM" src="https://github.com/iterative/dvclive/assets/2308172/612ec86f-6542-4435-b1db-d9dac38e69f3">

Reply:
pre-commit messed with the formatting